src/lib/Core.jsx

```diff
@@ -312,9 +312,10 @@ const Core = function ({
             />
           </div>
           <div>
+            <ProgressBar  currentQuestion={currentQuestionIndex} totalQuestions={questions.length}/>
             {appLocale.question}
             {' '}
-            {currentQuestionIndex + 1}
+            {currentQuestionIndex + 1} of {questions.length}
```

Comment:
prefer `/` rather than `of` so that we don't need to apply locale here.

Reply:
I was using / before you said you want question x of y  and you dont want progress bar?

Reply:
Sorry for the confusion. Please use `/` instead. For progress bar, I'd take Mrinmay's work since his work is more like what I want.

Reply:
@wingkwong ,I have updated and removes progress bar ,I wanted to say just one thing that i you can also tell me to modify progress bar according to you as i am contributor also ,i will take your changes and i have not made a bad progress bar,but you want  someone else work  for no reason

Reply:
The reasons are 
- Mrinmay's work is more close to my expectation in terms of quality and less changes to merge. 
- The UI of your progress bar is pretty bad tbh.
- Bad color contrast: red background with black text. 
- One shouldn't expect to see a progress like `0.14285714285%`
- Not configurable at all.

Reply:
my progress bar doesnt show 0.142857 ,you dont tell me to improve just reject me